src/Application.php

```diff
@@ -86,7 +86,7 @@ public function middleware($middlewareQueue)
         $middlewareQueue
             // Catch any exceptions in the lower layers,
             // and make an error page/response
-            ->add(ErrorHandlerMiddleware::class)
+            ->add(new ErrorHandlerMiddleware(null, Configure::read('Error')))
```

Comment:
This null is a bit awkward, but hard to remove now :cry:

Reply:
Yeah I don't like it either. Only way to avoid it is drop the first argument. Exception renderer can be set through config too.